### PR TITLE
added popup

### PR DESCRIPTION
**Pull request by Alissy27**
[Demo](https://alissy27.github.io/Test/) |
[Code base](https://github.com/Alissy27/Test/blob/main/index.html)

### CODE REVIEW

File: submissions/Alissy27/popup/index.html [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+<body>
+    
+<header class="header header__flex">
+    <img class="logo" src="https://alissy27.github.io/Test/puma.jfif" alt="puma">
+    <nav>
+        <ul class="nav">
+            <li class="nav__li"><a href="#">Mail</a></li>
+            <li class="nav__li"><a href="#">Images</a></li>
+            <li class="nav__li"><a href="#">
+                <img class="nav__img" src="https://alissy27.github.io/Test/img/bell.png" alt="bell"></a>
```
Comment (A-Ostrovnyy) [inline submissions/Alissy27/popup/index.html:19]: Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

File: submissions/Alissy27/popup/style.css [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+.nav{
+    display: flex;
+    align-items: center;
+}
+
+.nav__li{
+    margin-left: 15px;
+}
+
+.nav__popup label{
```
Comment (A-Ostrovnyy) [inline submissions/Alissy27/popup/style.css:36]: Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css) Also, avoid too specific CSS selectors. [Read more about that](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Please provide a link with a demo to validate your work properly.
Also, edit your first message according to the template.

File: submissions/Alissy27/popup/index.html (hunk 1 above)
Comment (Alissy27) [reply submissions/Alissy27/popup/index.html:19]: deleted alts from icons

File: submissions/Alissy27/popup/style.css (hunk 2 above)
Comment (Alissy27) [reply submissions/Alissy27/popup/style.css:36]: added class to label, changed css

**Comment by Alissy27** [comment]
[Demo](https://alissy27.github.io/Test/) |
[Code base](https://github.com/Alissy27/Test/blob/main/index.html)

File: submissions/Alissy27/popup/index.html [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+            </li>
+            <li class="nav__li nav__popup">
+                <input type="checkbox" id="popup__checkbox">
+                <label for="popup__checkbox" class="label__checkbox">
```
Comment (A-Ostrovnyy) [inline submissions/Alissy27/popup/index.html:23]: Avoid shifting sibling elements on hover or focus state. [Watch video](https://share.vidyard.com/watch/aBARHpHNS6CbRpQD4RpUVJ?)

File: submissions/Alissy27/popup/index.html [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Popup</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+<header class="header header__flex">
+    <img class="logo" src="https://alissy27.github.io/Test/puma.jfif" alt="puma">
```
Comment (A-Ostrovnyy) [inline submissions/Alissy27/popup/index.html:13]: A logo should be a link that leads to the home page.

File: submissions/Alissy27/popup/index.html (hunk 1 above)
Comment (A-Ostrovnyy) [inline submissions/Alissy27/popup/index.html:19]: Add `cursor: pointer` to all interactive elements.

File: submissions/Alissy27/popup/index.html [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+                                <a href="#">
+                                    <img class="popup__img" src="https://alissy27.github.io/Test/img/photos.png" alt="photos">
+                                    <p>Photos</p>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#">
+                                    <img class="popup__img" src="https://alissy27.github.io/Test/img/apps.png" alt="apps">
+                                    <p>Apps</p>
+                                </a>
```
Comment (A-Ostrovnyy) [inline submissions/Alissy27/popup/index.html:92]: Fix styles on focus state.
image

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
Please add links to your first message

**Pull request closed without merging**